tensorflow_lite_support/tools/Build_TensorFlow_Lite_Support_Libraries_with_Bazel_and_Colab.ipynb

```diff
@@ -0,0 +1,255 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        --config=monolithic \
</pre><p> </p><p>This is not needed as well. It's a flag used for tflite select ops, which is not needed here. A lot commands in our repo has this one, but Xunkai and I found it yesterday that it is not needed. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tflite-support/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
OK